R/unlockREDCap.R

```diff
@@ -49,23 +46,26 @@ key_saved <- function(envir, key)
 #' each key using getPass and store it in memory.
 #' 
 #' IMPORTANT: Make sure that R is set to NEVER save workspace to .RData
-#' as this is the equivalent of writing the API_KEY to a local
+#' as this *is* writing the API_KEY to a local
 #' file in clear text.
 #'
 #' @param connections character vector. A list of strings that define the
 #'          connections with associated API_KEYs to load into environment. Each
 #'          name should correspond to a REDCap project for traceability, but 
 #'          it can be anything. The variable name in the environment is this
 #'          name, or if a named vector the name associated.
-#' @param envir environment. The target environment for the data. Defaults to .Global
+#' @param envir environment. The target environment for the connections. Defaults to NULL
+#'          which returns the keys as a list. Use \code{\link{globalenv()}} to assign the
```

Comment:
Should be `\code{\link{globalenv}}` (without the parentheses). The link won't form correctly with the parentheses (causes a warning in check)